src/test.props

```diff
@@ -14,4 +14,10 @@
   <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Tests')) AND $(TestCategories.Contains('OuterLoop'))">
     <XunitOptions>-parallel none</XunitOptions>
   </PropertyGroup>
+  <ItemGroup>
+      <TestCommandLines  Condition="'$(TargetOS)'!='Windows_NT'" Include="sudo ./InstallRootCertificate.sh --service-host $(ServiceUri) --cert-file  ~/tmp/wcfra.crt"/>
+   </ItemGroup>
+  <Target Name="CopyInstallRootCertScript" Condition="$(MSBuildProjectName.EndsWith('.Tests'))" BeforeTargets="Build">
```

Comment:
Wondering if we need all the Tests folder - maybe just the ones that include "Security"?